src/gui/menu_list.cpp

```diff
@@ -0,0 +1,24 @@
+#include "gui/menu_item.hpp"
+#include "gui/menu_list.hpp"
+#include "gui/menu_manager.hpp"
+
+ListMenu::ListMenu(std::vector<std::string> items_, int* selected_) : 
```

Comment:
Please pass arguments as references to avoid an unneeded copy call, and as `const` if you don't modify the object for the caller.